backend/django_core/templates/user/register/_2.html

```diff
@@ -15,9 +15,7 @@
                 class="text-xs text-surface-300 md:text-[0.75vw]"
             >
                 {% if form.non_field_errors %}
-                    {% for error in form.non_field_errors %}
-                        {{ error }}
-                    {% endfor %}
```

Comment:
This is not the way `tattoo`. you show error specific to this field.

maybe with

```python
username.errors
```